### DISCUSSION TITLE
Does Oxalis support the Enhanced PEPPOL eDelivery network?

### DISCUSSION THREAD

**Discussion by canilsenlogiq** [OxalisCommunity/oxalis#633] [Q&A] [answered]
In a document released by the Norwegian PEPPOL authority, it says that since Oxalis 4.0, Oxalis has supported the Enhanced PEPPOL eDelivery network.

Ref.: https://anskaffelser.dev/payment/g1/docs/current/rulebook/#_oxalis
`As from version 4.0 Oxalis provides full support for the Enhanced Peppol eDelivery network.`

But I can find no other source or documentation of this.
Is this correct? Is there anywhere I can find some documentation of using Oxalis to send documents through the Enhanced Peppol eDelivery network? Or is it just the normal Oxalis flow, similar to how we send every other document through Oxalis?

**Comment by aaron-kumar** [comment, marked as answer]
@canilsenlogiq : Did you tried sending payment documents? These are supported out-of-box. 
It is normal flow as for other Peppol documents through Oxalis

**Reply by canilsenlogiq** [reply]
@aaron-kumar Thanks for answering. No, we never tried sending payment documents.
